app/views/entities/show.rb

```diff
@@ -83,6 +73,31 @@ def content
       end
     end
 
+    def entity_tab_items
```

Comment:
Nice and elegant, `%(field: "#{http://data.europeana.eu/#{api_path}}")` should be `%(field: "http://data.europeana.eu/#{api_path}}")`.